http/events.go

```diff
@@ -49,17 +49,20 @@ func handleEventsSubscribeWebsocket(args eventSubscribeArgs) (websocket.StatusCo
 		case message := <-ch:
 			logger.Debug("Sending message to websocket", "message", message)
 			var messageBytes []byte
+			var messageType websocket.MessageType
 			if args.json {
 				messageBytes, err = protojson.Marshal(message)
+				messageType = websocket.MessageText
+				messageBytes = []byte(string(messageBytes) + "\n")
```

Comment:
OOI why the newline character? Seems like it's more a client-side concern, as the data framing is already well defined by the websocket message.

Reply:
It's mostly for the convenience when using tools like `jq`, which work best when JSON is formatted into lines.